src/locator.c

```diff
@@ -426,8 +425,8 @@ int HAMLIB_API locator2longlat(double *longitude,
     /* For x(=longitude) and y(=latitude) */
     for (x_or_y = 0;  x_or_y < 2;  ++x_or_y)
     {
-        ordinate = -90.0;
-        divisions = 1;
+        double ordinate = -90.0;
+        int divisions = 1;
 
```

Comment:
Just a question, will Bill raise an issue that moving these declarations down into the code block will cause problems with some C89 compilers as he did a while back?